### PR TITLE
New Chaplain Nullrod

### PR DESCRIPTION
**Pull request by KeRSedChaplain**
## About The Pull Request
Adds in a new nullrod option for chaplains, the Divine Lightblade from citadel, which is a subtype of Dual Esword. Comparatively, less efficient of a murder tool. The sword deals 20 brute damage, has no AP, and a 30% chance to block both melee hits and projectiles, however, turning the blade on requires two hands, and slows your movement speed immutably (this means stims can't counteract it).

## Why It's Good For The Game
More nullrod variety is always neat, this is one of a few changes I'll be working on down the road to make more chaplain weapons feel less like reskins of one other weapon type.

## Proof Of Testing
I tested every change until I fixed every bug with the sword, the thing is fully functional for spreading FAITH.

## Changelog

:cl:
add: Ports the Divine Lightblade nullrod from citadel
/:cl:

### CODE REVIEW

**Comment by KeRSedChaplain** [comment]
I forgot about the feature freeze AUGH

please, your omnipotence...

File: modular_zubbers/code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -35,3 +35,145 @@
...
+///////////////////////////////
+//// HYPEREUTACTIC BLADE /////
+/////////////////////////////
+/obj/item/dualsaber/chaplain
+	icon = 'modular_zubbers/icons/obj/weapons/1x2.dmi'
+	icon_state = "hypereutactic"
+	worn_icon = 'modular_zubbers/icons/mob/clothing/belt.dmi'
+	inhand_icon_state = "hypereutactic"
+	lefthand_file = 'modular_zubbers/icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'modular_zubbers/icons/mob/inhands/64x64_righthand.dmi'
```
Comment (Majkl-J) [inline modular_zubbers/code/game/objects/items/holy_weapons.dm:48]:
```suggestion
	// Original from Citadel by Sishen1542
	icon = 'modular_zubbers/icons/obj/weapons/1x2.dmi'
	icon_state = "hypereutactic"
	worn_icon = 'modular_zubbers/icons/mob/clothing/belt.dmi'
	inhand_icon_state = "hypereutactic"
	lefthand_file = 'modular_zubbers/icons/mob/inhands/64x64_lefthand.dmi'
	righthand_file = 'modular_zubbers/icons/mob/inhands/64x64_righthand.dmi'
```

**Review by Majkl-J — APPROVED** [review]
Ill make this an exception to the freeze, just gotta ask you to leave some credit and it's all good

File: modular_zubbers/code/game/objects/items/holy_weapons.dm (hunk 1 above)
Comment (KeRSedChaplain) [reply modular_zubbers/code/game/objects/items/holy_weapons.dm:48]: Oh true, reminds me I should probably make a .txt credits file once I've compiled all the various clockwork cult changes I'm planning for after the freeze is lifted.

**Pull request merged**